sdk/storage/azure-storage-blob/azure/storage/blob/_shared/uploads.py

```diff
@@ -149,7 +159,6 @@ def __init__(
         self.parallel = parallel
 
         # Stream management
-        self.stream_start = stream.tell() if parallel else None
```

Comment:
We don't need this any more?

Reply:
This `self.stream_start` was not used anywhere as far as I could tell. It's definitely not used internally, and I couldn't see anywhere outside of the class it was being used. It's also a private internal class. Seems like it was left over from some old implementation?
I needed to remove it since the new wrapper stream I created for encryption doesn't support seeking or `tell`.